### DISCUSSION TITLE
New vehicle not added

### DISCUSSION THREAD

**Discussion by airmaxx** [teslamate-org/teslamate#2521] [Q&A]
From what I read, a new vehicle is being added automatically.

I've now driven 100km in 5 different drives, I restarted the docker container many times, yet the new car is not shown. Mobile access for the new car has also been activated. 

What can an do so that Teslamate knows about the new car - which is the second vehicle in my account.

**Comment by natrlhy** [comment]
Are both vehicles on the same Tesla account? I have both of ours on the same account. Right when I took delivery in the Tesla mobile app I restarted my TeslaMate docker containers and it picked it up right after

**Reply by cwanja** [reply]
If this does not work, and they are on the same Tesla account, log out and log back into TeslaMate. 

But assuming they are both on the same Tesla account, restarting the container should pick it up!

**Comment by airmaxx** [comment]
Thanks for all the tips. Logging out from Teslamate did the trick. Now both cars are in Teslamate visible.

**Comment by jjexpat00** [comment]
Similarly, signing out of Teslamate and reauthenticating with tokens resolved the issue where I accidentally removed a vehicle from my Tesla account and readded it.

**Comment by MrNickIE** [comment]
This is absolutely not working for me.  Logging in and out with new tokens etc.  Still no second vehicle, but both show on the app

**Reply by natrlhy** [reply]
I recall you have to stop all the containers and start them up again in order to see the new vehicle

**Reply by MrNickIE** [reply]
Thank you.  Actually completely my fault.  

The account that Tesla created for us was a new one, with both cars.  The Tesla Token app was still logged in with the old one, which only had access to the first car! 

Thanks for replying though, all working :)
